lib/config-all.json

```diff
@@ -253,6 +253,7 @@
     "requestanimationframe",
     "script/async",
     "script/defer",
+    "scroll-to-options",
```

Comment:
could you move the test into the css directory? I think its more fitting there

Reply:
@rejas  Why would you move it into the css directory? It is a test of JS function and its behaviour. Sorry for the pause, didn't read GitHub notifications.

Reply:
you're right, forget that :-)